examples/z_put.py

```diff
@@ -17,7 +17,7 @@
 import argparse
 import json
 import zenoh
-from zenoh import config, Value
+from zenoh import config, Value, Encoding
```

Comment:
It seems like Encoding is only used in commented code.
How about using `encoding=zenoh.Encoding.TEXT_JSON` instead?
BTW, maybe we can also remove `from zenoh import config, Value` because it's not used.